lib/discourse_slack/slack.rb

```diff
@@ -22,6 +22,12 @@ def self.format_channel(name)
       (name.include?("@") || name.include?("\#"))? name : "<##{name}>"
     end
 
+    def self.format_tags(names)
+      return "" unless if SiteSetting.tagging_enabled? && names.present?
```

Comment:
:grin: I already found that typo in local and fixed. Will update it soon with tests :+1: